### DISCUSSION TITLE
使用 package.tools.xmake 编译出的包的名称为 source

### DISCUSSION THREAD

**Discussion by q962** [xmake-io/xmake#5440] [Q&A]
我定义了一个本地的 package

我想用使用 `	import("package.tools.xmake").install(package)` 来自动编译。
只有一个 c 文件，项目提供的编译命令跨平台支持不到位。

但是结果的 lib 名是 source ，该名称是xmake 存放源码的目录名称。
应该使用 package name。

可以通过参数进行配置吗？

我还想问，install 接受的配置是不是 `xmake f` 命令接受的配置参数？

**Comment by star-hengxing** [comment]
包描述是构建系统无关的，构建系统描述还是要写的
https://github.com/xmake-io/xmake-repo/blob/dev/packages%2Fa%2Faseprite-tga%2Fxmake.lua

**Reply by q962** [reply]
明白了。那参数呢可以填写那些内容？

**Reply by star-hengxing** [reply]
用 tools.xmake 全局搜一下 xmake-repo 里的包
